src/app/entity-groups/research-entities/item-pages/org-unit/org-unit.component.html

```diff
@@ -32,6 +32,14 @@
     </ds-generic-item-page-field>
   </div>
   <div class="col-xs-12 col-md-7">
+    <ds-generic-item-page-field
```

Comment:
Do you considered creating a specific component?

Reply:
Thank you!

I think it's a good point @paulo-graca, Should we extend the base `src/app/item-page/simple/field-components/specific-field/item-page-field.component.ts` in order to create a new `ds-item-img-page-field` or similar ?

Reply:
I think you could turn this into a specific component that supports images/logos, as that might make it easier to work with.  It might be called `ds-item-page-img-field` (which is similar to other existing fields like `ds-item-page-uri-field`, etc).

All that said, honestly, I'm OK with this change either way.  It might be fine to leave this in the generic component to allow other fields to optionally display an image, e.g.  if you wanted a `ds-item-page-uri-field` with an image next to it.

So, I'd consider moving this to another component to be optional.